backend/gameplay/consumers.py

```
@@ -1145,6 +1148,25 @@ async def accept_challenge(self, data):
 
 		await challenge_obj.async_delete()
 
+	async def decline_challenge(self, data):
+		challenge_sender_username = data["challenge_sender"]
+		challenge_sender_user_obj: UserAuthModel = await UserAuthModel.async_get_user_model_from_username(challenge_sender_username)
+		challenge_sender_id = await challenge_sender_user_obj.async_get_player_id()
+
+		challenge_obj: GameChallenge | None = await GameChallenge.async_get_challenge_from_sender_username(challenge_sender_username)
+
+		if challenge_obj == None:
```

Ruff (0.11.9)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)